tests/migrations/test_operations.py

```diff
@@ -51,7 +51,7 @@ def make_test_state(self, app_label, operation, **kwargs):
         return project_state, new_state
 
     def set_up_test_model(
-            self, app_label, second_model=False, third_model=False,
+            self, app_label, second_model=False, third_model=False, index=False,
```

Comment:
I think testing multi-column indexes would ensure that single column indexes are also working fine. Won't they ?